extpoints/extpoints.go

```diff
@@ -0,0 +1,180 @@
+// generated by go-extpoints -- DO NOT EDIT
```

Comment:
@vishh - this is generated by go generate but is causing boilerplate check to fail. Should I add this file to .gitignore & not commit? I've added `go generate` to the Makefile so this will be created on each build. Guess we need to update travis config as well?